### DISCUSSION TITLE
Dismiss any screen that is presenting

### DISCUSSION THREAD

**Discussion by Cu-Toof** [pointfreeco/swift-composable-architecture#3322] [General]
I'm implementing a feature that receive notification and move to a screen.
I have a tabview that have 3 tab A B C
In tabview, when notification is received, move to tab B and remove all path.
But I'm in tab A or C and a screen is presenting. I have to dismiss this screen to show B

I used to Coordinator Pattern and deeplink is easy handled.
Please tell me if you have any way to handle deeplink from root view. 
Thanks

**Comment by mbrandonw** [comment]
Hi @Cu-Toof, it's hard to provide help without some code. Can you please build a very simple app that demonstrates your situation and shows the problem?
